setup.py

```diff
@@ -29,6 +29,9 @@
         'zope.interface',
         'zope.schema',
     ],
+    extras_require=dict(test=[
```

Comment:
Wir haben in `zeit.addcentral` und `zeit.wysiwyg` bereits extras_require gefunden und es daher übernommen. Sollten wir es dann dort auch entfernen? Oder lassen wir es jetzt stehen um mal den Anfang zu machen?